packages/expo-cli/lib/insert.js

```diff
@@ -35,7 +35,7 @@ Bugsnag.start();
 
 const getCode = async (projectRoot) => {
   const manifestRange = await detectInstalled(projectRoot)
-  const isPostV7 = !manifestRange || semver.satisfies('7.0.0', manifestRange)
+  const isPostV7 = !manifestRange || semver.ltr('6.99.99', manifestRange)
```

Comment:
What if someone used a range to skip 7 entirely for some reason?  E.g. `6.6.1 || >=8.0.0`.  We'd want v7 behaviour here but it would act as v6.

Is there some way to sidestep this entirely by cunning write a single piece of code to handle both scenarios?

Reply:
Ok, discussed in person and agreed that that is not a realistic scenario due to the nature of CLI use.